deployment/aks-periscope.yaml

```diff
@@ -108,30 +119,38 @@ data:
   AZURE_BLOB_ACCOUNT_NAME: # <accountName, base64 encoded>
   AZURE_BLOB_SAS_KEY: # <saskey, base64 encoded>
 ---
-apiVersion: v1 
-kind: ConfigMap 
+apiVersion: v1
```

Comment:
💡What is the change here?

Reply:
My guess is one of our IDEs probably was set to automatically trim whitespaces out of yamls. We can revert this change if you'd like